exercises/src/components/exercise/answer.jsx

```diff
@@ -7,16 +8,17 @@ import cn from 'classnames';
 //import _ from 'underscore';
 //import { AnswerActions, AnswerStore } from 'stores/answer';
 
+export default
```

Comment:
does this work? the whitespace is kind of gross but its probably auto generated so if it works /shrug

Reply:
yeah the new decorator pattern is that exports, then decorators, then definition.

There wasn't a codmod for that that I could find hence the regex in the update script